examples/seismic/model.py

```diff
@@ -366,12 +367,11 @@ def initialize_damp(damp, nbpml, spacing, mask=False):
         mask => 1 inside the domain and decreases in the layer
         not mask => 0 inside the domain and increase in the layer
     """
+    shape = tuple(i + 2*nbpml for i in damp.grid.subdomains['phydomain'].shape)
+    data = np.zeros(shape)
 
-    phy_shape = damp.grid.subdomains['phydomain'].shape
-    data = np.ones(phy_shape) if mask else np.zeros(phy_shape)
-
-    pad_widths = [(nbpml, nbpml) for i in range(damp.ndim)]
-    data = np.pad(data, pad_widths, 'edge')
+    if mask:
+        data[tuple(slice(nbpml, -nbpml) for i in shape)] = 1.
```

Comment:
`data = np.ones(shape) if mask else np.zeros(shape)` 
As it is damp initialized with 0 in the boundary layer instead of 1 for `mask`